docs/hooks/hooks.md

```diff
@@ -6,4 +6,1335 @@ sidebar_position: 4
 
 Scaffold-ETH 2 provides a collection of custom React hooks designed to simplify interactions with your deployed smart contracts. These hooks are wrappers around `wagmi`, automatically loading the necessary contract ABI and address. These hooks offer a simplified and streamlined interface for interacting with your smart contracts.
```

Comment:
I think we can change the text here too:

"These hooks are wrappers around Wagmi, an easy-to-use interface with typescript autocompletions for reading from, writing to, and monitoring events emitted by your smart contracts."

(from the SE2 landing)